README.md

```diff
@@ -28,6 +28,24 @@ for installing dependencies, you can refer to the CI scripts in this repo:
 - [install-debian-deps.sh](.github/scripts/install-debian-deps.sh) (Debian, Ubuntu, Mint, etc.)
 - [install-macos-deps.sh](.github/scripts/install-macos-deps.sh) (macOS with Homebrew)
 
+## running tests
+Tests are located in `test/` and are written using doctest (which is obtained via a wrap)
```

Comment:
this is wrong, just hadn't updated it yet (I'm well ahead on this PR at this point in a different branch trying to port some DSP code)